index.bs

```diff
@@ -1689,12 +1689,19 @@ An {{XRPose}} describes a position and orientation in space relative to an {{XRS
 <pre class="idl">
 [SecureContext, Exposed=Window] interface XRPose {
   [SameObject] readonly attribute XRRigidTransform transform;
+  [SameObject] readonly attribute DOMPointReadOnly? linearVelocity;
```

Comment:
"populate a pose" should be populating these somehow

Reply:
Thanks! 
I couldn't find where that happened. I will update.